### PR TITLE
Tile gradient for new Ultimate On Demand tiles

### PR DESCRIPTION
**Pull request by Salstar24**
## Description
New gradient colours (from dark blue to teal) for the 'Ultimate On Demand' tiles (the new Netflix channel page under /watch/channel/ultimate-on-demand).

## Related Issue
Closes https://github.com/sky-uk/digital-engagement/issues/1092
https://github.com/sky-uk/toolkit/issues/467

## Motivation and Context
This is part of the Netflix launch so is needed for tomorrow (1st November).

## How Has This Been Tested?
Manually via the local 'preview' command on Toolkit.

Here is a gif of the result
image

This has been confirmed by Tom Postlethwaite

Colours for reference:
0% we have 000514, at 50% we have 133e73 and at 100% we have 3c7191 all of which should be at 100% opacity

## Types of Changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [ ] Internal *(framework-only)*
- [ ] Bug Fix *(non-breaking change which fixes an issue)*
- [X] New Feature *(non-breaking change which adds functionality)*
- [ ] Breaking change *(fix or feature that would cause existing functionality to change)*

## Browser Support

- [ ] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE9
- [ ] IE10
- [ ] IE11
- [ ] Opera
- [ ] Safari
- [ ] Mobile Devices
- [ ] Screen-readers (e.g. JAWS, Apple VoiceOver)

## Checklist

- [X] All **design and code** conforms to the [Design Contribution Guidelines](https://github.com/sky-uk/toolkit/blob/develop/CONTRIBUTING.md#design-contributions).
- [ ] New functions and mixins have appropriate tests.
- [X] All new and existing tests passed.
- [ ] I have added instructions on how to test my changes.

### CODE REVIEW

**Comment by skitson** [comment]
If this is signed off to go into the Toolkit gradients then it looks good. If not, you might need to use the `map-merge()` SCSS function in your Sky Pages or Pages App application.

Something like this:

```
$new-gradients: (
    ultimate-on-demand: (
        0%:   #193769,
        100%: #1ea0be
    )
);

$gradients: map-merge($gradients, $new-gradients);
```

**Comment by Salstar24** [comment]
@skitson Thank you for having a look. Who would need to sign off these gradient values? I know they've come directly from Ents marketing.

**Comment by skitson** [comment]
@Salstar24 I'm really not sure to be honest apart from suggesting you have a chat with @mikejgregory. He's Lead Designer.

I'd suggest that if you want to get this in quickly (I presume you do) then you should put it into your app for now. The Toolkit release process will probably slow you down at this point.

**Comment by MrDinsdale** [comment]
If this is urgently needed I'd go with @skitson's suggestion of adding to your Scss directly. Although on the last line you might need to use `$gradients: map-merge($gradients, $new-gradients) !global;`.

**Comment by mibu5** [comment]
@MrDinsdale, @steveduffin, @skitson The gradient colous has been appried by Tom Postlethwaite, can you please take another look the the PR, please.

image

File: packages/sky-toolkit-core/settings/_gradients.scss [hunk 1]
```diff
@@ -80,6 +80,11 @@ $gradients: (
     0%:   #000,
     100%: #262626
   ),
+  ultimate-on-demand: (
+    0%:   #000514,
+    50%:   #133e73,
```
Comment (lukewhitehouse) [inline packages/sky-toolkit-core/settings/_gradients.scss]: can we remove the extra space before the hex code here. Sorry I know its pedantic!

**Comment by lukewhitehouse** [comment]
Happy to approve once the above is done

File: packages/sky-toolkit-core/settings/_gradients.scss (hunk 1 above)
Comment (mibu5) [reply packages/sky-toolkit-core/settings/_gradients.scss]: Done! 👍

**Comment by lukewhitehouse** [comment]
Before we merge the commit messages will need squashing an formatting into our usual setup: `[refs #000] your PR summary here`

**Comment by mibu5** [comment]
@lukewhitehouse We've squashed commits and referenced the issue. 👍

**Comment by Salstar24** [comment]
@MrDinsdale @lukewhitehouse @StefanMcCready I guess given the chaos today this release might not be happening?

Just let us know when you're back up and running and when the merge might be. :)

**Pull request merged**